skillserver/skillserver.go

```diff
@@ -69,10 +68,14 @@ func SetRootPrefix(prefix string) {
 }
 
 // Run will initialize the apps provided and start an HTTP server listening on the specified port.
-func Run(apps map[string]interface{}, port string) {
+// Use an optional fourth boolean parameter to indicate whether to skip AWS cert's validation
+func Run(apps map[string]interface{}, port string, insecureSkipVerify ...bool) {
 	router := mux.NewRouter()
-	initialize(apps, router)
-
+	isv := false
+	if len(insecureSkipVerify) > 0 {
+		isv = insecureSkipVerify[0]
+	}
+	initialize(apps, router, isv)
```

Comment:
Rather than an optional bool to multiple functions I think the better solution which would require less changes and be and more idiomatic with the existing code would be to implement an additional Set function such as `SetRootPrefix` to set the InsecureSkipVerify option.

Reply:
ok, I will give it a try.